backend/bin/deal-observer-backend.js

```diff
@@ -57,10 +56,10 @@ const observeActorEventsLoop = async (makeRpcRequest, pgPool) => {
       Sentry.captureException(e)
     }
     const dt = Date.now() - start
-    console.log(`Loop "${LOOP_NAME}" took ${dt}ms`)
+    console.log(`Loop "Observe actor events" took ${dt}ms`)
 
     if (INFLUXDB_TOKEN) {
-      recordTelemetry(`loop_${slug(LOOP_NAME, '_')}`, point => {
+      recordTelemetry('loop_builtin_actor_events', point => {
```

Comment:
Changing this to "loop_observe_actor_events" was unintentional, @NikolasHaimerl and me agreed that the `LOOP_NAME` is risky